conf/nginx.conf

```diff
@@ -0,0 +1,23 @@
+server {
+    listen 80 default_server;
+    charset utf-8;
+    root /usr/share/nginx/html;
+    index index.html;
+
+    # Handle /_status endpoint with a 200 response once the container is up
+    location = /_status {
+        add_header Content-Type text/plain;
+        return 200 'Success!';
+    }
+
+    # When requesting static paths with known extensions, try them, and return a 404 if not found
+    location ~* .+\.(css|js|html|png|jpe?g|gif|bmp|ico|json|csv|otf|eot|svg|svgz|ttf|woff|woff2|ijmap|pdf|tif|map) {
```

Comment:
BUT WHAT ABOUT `TIFF`?! No seriously, is this regexp reasonable for maintenance?

Reply:
Haha. Yeah, I think you have a good point there.

I actually took this from an old project where I first used this kind of approach and didn't stop to think it through too much.

I think just checking if it's a URL with an extension would be enough to safely assume it's a static file URL, without checking for specific known extensions.

Of course, if at some point we use dots in dynamic routes, it might not work, but I suppose that's not very likely.